### PR TITLE
fixed https://github.com/NuGet/Home/issues/1163

### PR DESCRIPTION
**Pull request by zhili1208**
https://github.com/NuGet/Home/issues/1163
removed all those
Merging in runtimes defined in ...
Restoring packages for ...
Resolving conflicts for ...
Checking compatibility for ...

the new log will becomes 

capture

### CODE REVIEW

**Comment by emgarten** [comment]
Lets fix this by filtering them at the console. It would be nice to keep these for debug purposes.
:watch:

File: src/NuGet.Commands/RestoreCommand.cs [hunk 1]
```diff
@@ -154,7 +154,7 @@ public Task<RestoreResult> ExecuteAsync()
                 return Enumerable.Empty<RestoreTargetGraph>();
             }
 
-            _log.LogInformation(Strings.FormatLog_RestoringPackages(_request.Project.FilePath));
```
Comment (emgarten) [inline src/NuGet.Commands/RestoreCommand.cs:157]: Leaving this one

File: src/NuGet.Commands/RestoreCommand.cs [hunk 2]
```diff
@@ -748,7 +748,7 @@ private void CheckDependencies(RestoreTargetGraph result, IEnumerable<LibraryDep
             var resultGraphs = new List<Task<RestoreTargetGraph>>();
             foreach (var runtimeName in runtimeIds)
             {
-                _log.LogInformation(Strings.FormatLog_RestoringPackages(FrameworkRuntimePair.GetName(graph.Framework, runtimeName)));
+                _log.LogDebug(Strings.FormatLog_RestoringPackages(FrameworkRuntimePair.GetName(graph.Framework, runtimeName)));
```
Comment (emgarten) [inline src/NuGet.Commands/RestoreCommand.cs]: Leave this also for the UI

**Comment by emgarten** [comment]
Add back in the restore messages then :shipit:

**Comment by deepakaravindr** [comment]
:shipit: after making the changes you suggested. Also, ensure that the behavior of 'nuget.exe restore <project.json>' for default VerbosityLevel is intact

**Comment by yishaigalatzer** [comment]
@zhili1208 why is this PR still open?

**Pull request closed without merging**